src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -678,4 +665,18 @@ private static JSONObject getSourceObject(final TravelOptions travelOptions,
         }
         return source;
     }
+
+    /**
+     * Parse a geometry into a json object
+     * @param geometry The Geometry to be parsed
+     * @throws JSONException If failing to add any of the geometry qualities
+     */
+    private static JSONObject parseGeometry(Geometry geometry) throws JSONException {
```

Comment:
very minor: I wouldn't call that method `parse` because it is actually doing the opposite. maybe `createGeometryJson` or `generateGeometryJsonObject`?